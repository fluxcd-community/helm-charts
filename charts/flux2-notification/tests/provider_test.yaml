suite: test provider deployment
templates:
  - provider.yaml
tests:
  - it: should be empty by default
    asserts:
      - hasDocuments:
          count: 0
  - it: should match snapshot
    asserts:
      - matchSnapshot: {}
    set:
      providerlist:
      - name: on-call-slack
        spec:
          type: slack
          channel: general
          username: user1
          address: https://hooks.slack.com/services/YOUR/SLACK/WEBHOOK
          proxy: https://proxy.corp:8080
          secretRef:
            name: webhook-url
          certSecretRef:
            name: certref
