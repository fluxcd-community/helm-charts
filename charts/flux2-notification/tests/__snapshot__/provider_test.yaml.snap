should match snapshot:
  1: |
    apiVersion: notification.toolkit.fluxcd.io/v1beta1
    kind: Provider
    metadata:
      labels:
        app.kubernetes.io/instance: NAMESPACE
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/part-of: flux
        app.kubernetes.io/version: 0.31.3
        helm.sh/chart: flux2-notification-1.0.0
      name: on-call-slack
      namespace: NAMESPACE
    spec:
      address: https://hooks.slack.com/services/YOUR/SLACK/WEBHOOK
      certSecretRef:
        name: certref
      channel: general
      proxy: https://proxy.corp:8080
      secretRef:
        name: webhook-url
      type: slack
      username: user1
